recipes/gmtl/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.8)
+
+project(test_package C) # if the project is pure C
+project(test_package CXX) # if the project uses c++
+
+find_package(gmtl REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+# don't link to ${CONAN_LIBS} or CONAN_PKG::package
+target_link_libraries(${PROJECT_NAME} PRIVATE gmtl::gmtl)
+# In case the target project need a specific C++ standard
+target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_14)
```

Comment:
Is this minimum C++ standard correct? If so, you'll need to add a `validate()` method in the conanfile.py to require that C++ standard.

Reply:
Thanks for the hint. I'll check